apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -532,6 +532,34 @@ export default class CoreLibrary {
     }
   }
 
+  everySecondsEvent(inputEvent) {
+    if (inputEvent.args.unit === 'seconds') {
+      const previousTime = inputEvent.previousTime || 0;
+      const worldTime = this.getSecondsSinceReset();
+      // Repeat every n seconds
+      const moddedWorldTime = worldTime % inputEvent.args.n;
+      inputEvent.previousTime = moddedWorldTime;
+      // There are many ticks per second, but we only want to fire the event once (on the first tick where
+      // the time matches the event argument)
+      if (moddedWorldTime === 0 && previousTime !== 0) {
+        // Call callback with no extra args
+        this.eventLog.push(`atTime: ${inputEvent.args.n}`);
```

Comment:
`atTime` should be updated to a new label to differentiate it from `atTimeEvent`.